manga compiling script.py

```diff
@@ -1,92 +0,0 @@
-import os
-import shutil
-import tempfile
-from pathlib import Path
-

```

Comment:
Might want to rename your file with underscores _ instead of whitespace, it can mess things up pretty often